functests/config.go

```diff
@@ -0,0 +1,271 @@
+package functests
+
+import (
+	"time"
+
+	cincinnativ1beta1 "github.com/openshift/cincinnati-operator/pkg/apis/cincinnati/v1beta1"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	customResourceName = "example-cincinnati"
+	operatorName       = "cincinnati-operator"
+	operatorNamespace  = "openshift-cincinnati"
+	defaultNamespace   = "default"
+	crdName            = "cincinnatis.cincinnati.openshift.io"
+	resource           = "cincinnatis"
+	replicas           = 1
+	retryInterval      = time.Second * 5
+	timeout            = time.Second * 1200
```

Comment:
I think 20 minutes is too long to wait for the deployment to come up. I think 5 minutes should be enough time.

Reply:
Done